packages/developer-portal/src/components/analytics/costs/download-resources-table.tsx

```diff
@@ -80,7 +80,7 @@ export const DownloadResourcesTable: FC<DownloadResourcesTableProps> = ({ billin
             ],
             ctaContent: {
               onClick: handleDownloadCSV(billing, error),
-              icon: 'downloadSystem',
+              icon: 'downloadSolidSystem',
```

Comment:
Feel like this is a bit OTT but seeing as this is an existing icon (not sure when added) but would this constitute a breaking change and require a major version change so not to conflict with those currently using this icon? Unless it's recently added then we might get away with making it a minor change release

Reply:
It is an existing icon, yes